### PR TITLE
Count of images is now updated when images removed using "Rearrange Images" option

### LINKED ISSUE #605: Count of images doesn't update when images removed from 'Rearrange Images' option
Count of images doesn't update when images removed from 'Rearrange Images' Enhancement option

### Steps to reproduce
* Images to PDF
* Select Images 
* Select `Rearrange Images` option from More Enhancement options
* Remove some images 
* press `Back`

### Expected behaviour
Count of file ` x images(s) selected` should update which is there at bottom of `ImagesToPDF Fragment` 
### Actual behaviour
Files get removed but count of file ` x images(s) selected` doesn't update

### PR DESCRIPTION
**Pull request by SupreethBaliga**
# Description

Initially, when images were removed using "Rearrange Images" option, it still displayed the initial number of images. This issue is now fixed.

Fixes #605 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Tue Jan 08 21:49:33 IST 2019
+#Fri Mar 22 14:57:39 IST 2019
```
Comment (vyankatesh24) [inline gradle/wrapper/gradle-wrapper.properties]: Don't Change this file. Revert changes you have made.

File: build.gradle [hunk 2]
```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```
Comment (vyankatesh24) [inline build.gradle]: Don't Change this file. Revert changes you have made.

**Comment by vyankatesh24** [comment]
@SupreethBaliga Add issue no. in PR Template, in front of `Fixes #(issue)`

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (SupreethBaliga) [reply gradle/wrapper/gradle-wrapper.properties]: I actually did not make these changes manually. I don't know how they came up.

**Comment by SupreethBaliga** [comment]
Why is it showing that the checks have failed after I reverted those changes?

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (vyankatesh24) [reply gradle/wrapper/gradle-wrapper.properties]: Revert all changes in this file.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```
Comment (vyankatesh24) [inline gradle/wrapper/gradle-wrapper.properties]: Revert this change also.
After that we can merge it :+1:
